src/external_acl.cc

```diff
@@ -1139,32 +1143,13 @@ externalAclShutdown(void)
     }
 }
 
-ExternalACLLookup ExternalACLLookup::instance_;
-ExternalACLLookup *
-ExternalACLLookup::Instance()
-{
-    return &instance_;
-}
-
-void
-ExternalACLLookup::checkForAsync(ACLChecklist *checklist)const
-{
-    /* TODO: optimise this - we probably have a pointer to this
-     * around somewhere */
-    ACL *acl = ACL::FindByName(AclMatchedName);
-    assert(acl);
-    ACLExternal *me = dynamic_cast<ACLExternal *> (acl);
-    assert (me);
-    ACLExternal::ExternalAclLookup(checklist, me);
```

Comment:
This is that problematic FindByName() call that was blocking ACL refcounting. This PR addresses the above optimization TODO as well, but the related speedup ought to be negligible in most environments, and it is not the focus of this PR.